provision/ansible/playbooks/tasks/start-sync-gateway.yml

```diff
@@ -8,9 +5,6 @@
 - name: change permissions of sync gateway executable
   file: path=/home/sync_gateway/sync_gateway mode=a+x
 - debug: msg="Starting sync_gateway with config {{ sync_gateway_config_filepath }}"
```

Comment:
Could we also pull these steps out into the install-source and install-package script as well?